js/reference.js

```diff
@@ -3162,6 +3162,25 @@ var ERMrest = (function(module) {
         },
 
         /**
+         * The unique identifier for this tuple composed of the values for each
+         * of the shortest key columns concatenated together by an '_'
+         *
+         * @type {string}
+         */
+        get uniqueId() {
+            if (!this._uniqueId) {
```

Comment:
Change this check to `this_uniqueId === undefined` to be consistent with other apis.